Libraries/src/net/relinc/libraries/sample/LoadDisplacementSampleResults.java

```diff
@@ -13,20 +16,58 @@
 	public double[] load;
 	public double[] displacement;
 
-	public DataLocation loadDataLocation;
-	public DataLocation displacementDataLocation;
+	private DataLocation loadDataLocation;
+	private DataLocation displacementDataLocation;
 
-	Sample sample;
+	Sample sample; // super marginal to have references that are cycles. Sample references this class, which references Sample.
```

Comment:
could this be final? any way to make this safer?

Reply:
it's not that it changes that is the problem, I've just seen the case where it's `sample.results.sample` which is just meh